setup.py

```diff
@@ -9,17 +9,18 @@
     readme = readme_file.read()
 
 requirements = [
-    "aicspylibczi>=2.6.0",
-    "dask[array]>=2.9.0",
-    "fsspec>=0.8.4",
-    "numpy>=1.16",
-    "imagecodecs>=2020.5.30",
-    "imageio[ffmpeg]>=2.9.0",
-    "readlif>=0.2.1",
-    "lxml>=4.5.2",
+    "aicspylibczi~=2.7.5",
+    "dask[array]~=2.9.0",
+    "fsspec~=0.8.4",
+    "imagecodecs~=2020.5.30",
+    "imageio[ffmpeg]~=2.9.0",
+    "lxml~=4.5.2",
+    "numpy~=1.16",
+    "ome-types~=0.2.3",
```

Comment:
i am looking forward to when we can remove vendor/omexml.py  !

Reply:
What do you mean? Where I am living, it's already gone! :upside_down_face: